### PR TITLE
Update Scalingo profile

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: profiles/scalingo.json [hunk 1]
```diff
@@ -1,17 +1,20 @@
 {
   "name": "Scalingo",
-  "revision": "2015-07-22",
-  "vendor_verified": "2015-04-17",
+  "revision": "2016-07-22",
+  "vendor_verified": "2015-02-23",
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: Why 2015-03-23 and not to the current revision?

File: profiles/scalingo.json [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-  "status_since": "2015-04-17",
+  "status_since": "2015-02-03",
   "type": "Generic",
   "hosting": {
     "public": true,
     "private": true
   },
   "pricing": [
     {
+      "model": "free"
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: A free trial is not allowed here. Only if there really is a free plan.

File: profiles/scalingo.json [hunk 3]
```diff
@@ -44,23 +47,18 @@
       "language": "java",
       "versions": [
         "1.7.*",
+        "1.8.*",
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: This is duplicated now?! (1.8.*)

File: profiles/scalingo.json [hunk 4]
```diff
@@ -44,23 +47,18 @@
...
         "0.10.*",
-        "0.12.*"
-      ]
-    },
-    {
-      "language": "io.js",
-      "versions": [
-        "1.1.*"
+        "0.12.*",
+        "4.x",
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: Please use * instead of x for wildcards.

File: profiles/scalingo.json [hunk 5]
```diff
@@ -250,44 +260,43 @@
         "description": "",
         "type": "datastore",
         "versions": [
-          "2.6.10"
+          "3.2.x"
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: Please use * instead of x

File: profiles/scalingo.json [hunk 6]
```diff
@@ -250,44 +260,43 @@
...
         ]
       },
       {
         "name": "mysql",
         "description": "",
         "type": "datastore",
         "versions": [
-          "5.5.41"
+          "5.5.x"
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: Please use * instead of x

File: profiles/scalingo.json [hunk 7]
```diff
@@ -250,44 +260,43 @@
...
         ]
       },
       {
         "name": "postgresql",
         "description": "",
         "type": "datastore",
         "versions": [
-          "9.3.6"
+          "9.4.x"
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: Please use * instead of x

File: profiles/scalingo.json [hunk 8]
```diff
@@ -250,44 +260,43 @@
...
         ]
       },
       {
         "name": "redis",
         "description": "",
         "type": "datastore",
         "versions": [
-          "2.8.21"
+          "3.x"
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: Please use * instead of x

File: profiles/scalingo.json [hunk 9]
```diff
@@ -250,44 +260,43 @@
...
         ]
       },
       {
         "name": "elasticsearch",
         "description": "",
         "type": "datastore",
         "versions": [
-          "1.4.5"
+          "1.7.x"
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: Please use * instead of x

**Comment by stefan-kolb** [comment]
Hi @Soulou 

Thanks for your PR :smile: 
We need to fix a few comments before I can merge this in.

Cheers, Stefan

File: profiles/scalingo.json [hunk 10]
```diff
@@ -150,6 +152,13 @@
       ]
     },
     {
+      "name": "play",
+      "runtime": "scala",
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: Please use `java`as runtime for Play.

**Comment by stefan-kolb** [comment]
@Soulou Would you mind taking a look at the comments please? :smile:

**Comment by Soulou** [comment]
Apologies, my github notifications got flooded, I'm handling all that now.

File: profiles/scalingo.json (hunk 1 above)
Comment (Soulou) [inline profiles/scalingo.json]: Actually, my question is: what do you expect exactly in `vendor_verified` ?

Comment (Soulou) [inline profiles/scalingo.json]: Ok, go it from the doc, I'm updating it

Comment (Soulou) [inline profiles/scalingo.json]: 2015-02-23 → Date when scalingo.json has been added, it seems correct isn't it?

File: profiles/scalingo.json (hunk 2 above)
Comment (Soulou) [inline profiles/scalingo.json]: Ok removing it

File: profiles/scalingo.json (hunk 3 above)
Comment (Soulou) [inline profiles/scalingo.json]: Typo of course, it's 1.9

**Comment by Soulou** [comment]
Updated.

File: profiles/scalingo.json [hunk 11]
```diff
@@ -48,17 +45,17 @@
       "versions": [
         "1.7.*",
         "1.8.*",
-        "1.8.*"
+        "1.9.*"
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: There's no Java 1.9 right now. 1.8 seemed to be alright!

File: profiles/scalingo.json (hunk 1 above)
Comment (stefan-kolb) [inline profiles/scalingo.json]: `vendor_verified` means the last date an official member of the provider reviewed the profile.

File: profiles/scalingo.json [hunk 12]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "Scalingo",
-  "revision": "2015-07-22",
-  "vendor_verified": "2015-04-17",
+  "revision": "2016-08-04",
+  "vendor_verified": "2015-02-23",
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: `vendor_verified` means the last date an official member of the provider reviewed the profile. So this should be today?!

File: profiles/scalingo.json [hunk 13]
```diff
@@ -1,10 +1,10 @@
...
   "url": "https://scalingo.com",
   "status": "production",
-  "status_since": "2015-04-17",
+  "status_since": "2015-02-03",
```
Comment (stefan-kolb) [inline profiles/scalingo.json]: `status_since` means the date since Scalingo is in `production` Why did you change this?

**Comment by stefan-kolb** [comment]
@Soulou Just two minor comments and then we can merge this :smile:
